### PR TITLE
fix: Remove local realignment for TGA

### PR DESCRIPTION
**Pull request by mathiasbio**
### This PR:

This PR attempts to solve this bug: https://github.com/Clinical-Genomics/BALSAMIC/issues/1271 by removing realignment step for TGA. 

Removed: Realignment for TGA workflows.

unitedbeagle DAG:
image

civilsole DAG:
image

Running tests to verify integrity of workflows. Check boxes when workflow finishes without errors:
- [x] TGA T/N case unitedbeagle
- [x] WGS T case civilsole

### Review and tests: 
- [x] Tests pass
- [x] Code review
- [x] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: BALSAMIC/snakemake_rules/align/bam_compress.rule [hunk 1]
```diff
@@ -46,3 +46,49 @@ rule bam_compress_normal:
...
+        housekeeper_id= {"id": "{sample}", "tags": "normal"}
+    threads:
+        get_threads(cluster_config, "bam_compress")
+    message:
+        "Compressing bam to cram for {params.sample_id}"
+    shell:
+        """
+samtools view -h -T {input.fasta} --threads {threads} -C -o {output.cram} {input.bam}; 
+samtools index {output.cram}; 
+        """
```
Comment (ivadym) [inline BALSAMIC/snakemake_rules/align/bam_compress.rule:94]: EOF

File: BALSAMIC/snakemake_rules/align/bam_compress.rule [hunk 2]
```diff
@@ -46,3 +46,49 @@ rule bam_compress_normal:
 samtools view -h -T {input.fasta} --threads {threads} -C -o {output.cram} {input.bam}; 
 samtools index {output.cram}; 
         """
+
+rule bam_compress_tumor_wgs:
```
Comment (ivadym) [inline BALSAMIC/snakemake_rules/align/bam_compress.rule:50]: I'm not sure which is the best strategy here: defining an input file variable depending on the analysis type (WGS vs. targeted) or having code duplication in these rules. I would say that an if statement is better, mainly for maintainability, but feel free to disagree.

**Review by ivadym — APPROVED** [review]
Looks good 👍 🥇 

Left you just a minor comment 🧂

File: BALSAMIC/snakemake_rules/align/bam_compress.rule (hunk 2 above)
Comment (mathiasbio) [reply BALSAMIC/snakemake_rules/align/bam_compress.rule]: No it's a good idea! I didn't think for very long before making this change. I think I have a better and pretty obvious solution now :) Just waiting for the test-runs to finish!

Comment (ivadym) [reply BALSAMIC/snakemake_rules/align/bam_compress.rule]: Nice ⭐

**Pull request merged**